### PR TITLE
Send email to Reviewers

### PR DESCRIPTION
**Pull request by davidmoore1**

### CODE REVIEW

File: source/OptimaJet.DWKit.StarterApplication/Templates/ReviewProduct.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿@{
+    DisableEncoding = true;
+}
+<html>
```
Comment (chrisvire) [inline source/OptimaJet.DWKit.StarterApplication/Templates/ReviewProduct.txt]: We are going to have to figure out what looks good as an HTML email.  Maybe there are libraries???

File: source/OptimaJet.DWKit.StarterApplication/Migrations/20190201190247_AddReviewerLocale.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace OptimaJet.DWKit.StarterApplication.Migrations
+{
+    public partial class AddReviewerLocale : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<string>(
+                name: "Locale",
```
[the remark this answers is not in the document]
Comment (davidmoore1) [reply source/OptimaJet.DWKit.StarterApplication/Migrations/20190201190247_AddReviewerLocale.cs]: I was waiting until this was included to add it.

File: source/OptimaJet.DWKit.StarterApplication/Services/Workflow/WorkflowProductActionProvider.cs [hunk 3]
```diff
@@ -238,6 +239,16 @@ private async Task SendOwnerNotificationAsync(ProcessInstance processInstance, W
             }
         }
 
+        private async Task SendReviewerLinkToProductFilesAsync(ProcessInstance processInstance, WorkflowRuntime runtime, string actionParameter, CancellationToken token)
+        {
+            using (var scope = ServiceProvider.CreateScope())
+            {
+                var productRepository = scope.ServiceProvider.GetRequiredService<IJobRepository<Product, Guid>>();
+                Product product = await GetProductForProcess(processInstance, productRepository);
+                BackgroundJobClient.Enqueue<SendEmailService>(s => s.SendProductReviewEmail(product.Id, actionParameter));
```
[the remark this answers is not in the document]
Comment (davidmoore1) [reply source/OptimaJet.DWKit.StarterApplication/Services/Workflow/WorkflowProductActionProvider.cs]: Made change to convert to dictionary here and validate that "types" entry exists or throws an exception

**Comment by chrisvire** [comment]
@davidmoore1 I think this may be ready to merge.

**Pull request merged**